kpet/data.py

```diff
@@ -16,6 +16,8 @@
 import os
 from kpet.schema import Invalid, Struct, Reduction, NonEmptyList, \
     List, Dict, String, Regex, ScopedYAMLFile, YAMLFile, Class, Boolean, Int
+from kpet.schema import Succession
+
```

Comment:
Can you just add this to the list directly above?

Reply:
I could, but doesn't that make diffs more confusing? When each imported thing has its own line, it's clearer to see what was added or removed.

Reply:
Yeah, makes sense, go ahead. I just don't want a ton of lines, for this, but if it's easier for you, sure. I just do `git diff --color-words=.` for intra-line differences.